tests/test_all_protocols_binary_field.py

```diff
@@ -263,12 +260,12 @@ def run_server():
     time.sleep(0.2)
 
 
-@pytest.mark.skipif(cybin is None, reason="Must be run in cpython")
+@pytest.mark.skipif(_compat.PYPY, reason="Must be run in cpython")
```

Comment:
Because we are missing a `__init__.py` in the `cybin` folder, modern version of Python (not Python2) will treat this folder as a namespace module, so it's not `None`.